ddl/ddl_api.go

```diff
@@ -5252,7 +5252,7 @@ func buildPlacementSpecs(specs []*ast.PlacementSpec) ([]*placement.RuleOp, error
 			switch spec.Tp {
 			case ast.PlacementAdd:
 				rule.Action = placement.RuleOpAdd
-			case ast.PlacementAlter:
+			case ast.PlacementAlter, ast.PlacementDrop:
 				rule.Action = placement.RuleOpAdd
```

Comment:
When do we use `RuleOpDel`? Why do we need this `switch` snippet?

Reply:
`RuleOpDel` is at L5272. DDL stmts are not that closely related to PD api. Alter need to make use of both `RuleOpAdd` and `RuleOpDel`. This line is prepared for later half part - placement rules building. But drop will use continue to skip building and use `RuleOpDel` only.

Switch is used to make things more clear initially. But, yes, now it looks like a nested if-else statement should do the work, too.
It is unexpected that drop and alter can share same code.
I am OK on both switch and if-else, do I need to change to if-else?